R/kernel.r

```diff
@@ -219,7 +219,6 @@ shutdown = function(request) {
 
 initialize = function(connection_file) {
     connection_info <<- fromJSON(connection_file)
-    print(connection_info)
```

Comment:
Jupyter/IPython doesn't currently pass any such flags to kernels, though. So for now, let's just edit in debugging prints where we need them. It's been a long time since I needed to refer to this information for debugging, anyway.